README.md

```diff
@@ -1,70 +1,74 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Crypto-Metrics-Webapp
 
-## Available Scripts
+> Crypto-Metrics-Webapp: This is a mobile web application used to check a list of crypto metrics. It is created using the coinstats API. The webapp have several pages which includes the "Home page" with a list of items that could be filtered by some parameters; in the project, it's a list of metrics that can be filtered by the coin name and a "Details page" with more information about a selected coin.
 
-In the project directory, you can run:
+## Overview
 
-### `npm start`
+### Screenshots
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+## Live Demo
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+[link to live demo on Netlify]
```

Comment:
- [ ] You are expected to deploy your app and add the link here. Kindly do that.